playbooks/templates/.env.j2

```diff
@@ -107,64 +133,96 @@ AIRTABLE_FIELD={{ webportal_common_config.airtable_field }}
 # MongoDB
 
 # MongoDB username
-SKYNET_DB_USER={{ mongodb_config.skynet_db_user }}
+#
+# OPTIONAL: manually defined by user, or default will be used
+SKYNET_DB_USER={{ mongodb_config.skynet_db_user | default('admin') }}
 
 # MongoDB password
+#
+# REQUIRED: Ansible automatically generated 
+# TODO: Ansible doesn't not currently automatically generate this but it should 
 SKYNET_DB_PASS={{ mongodb_config.skynet_db_pass }}
 
 # MongoDB address or container name
+#
+# OPTIONAL: manually defined by user, or default will be used
+# TODO: Do these need to be able to be defined by the user?
```

Comment:
I don't think we *need* to make this configurable. Fewer configuration options might actually be better at this stage.